helm_charts/media/values.yaml

```
@@ -37,27 +37,33 @@ securityContext: {}
-        pathType: Prefix
+  #   - host: "sabnzb.neilwylie.com"
+  #     paths:
+  #     - path: /
+  #       pathType: Prefix
+  #   - host: "sonarr.neilwylie.com"
+  #     paths:
+  #     - path: /
+  #       pathType: Prefix
     - host: "couchpotato.neilwylie.com"
       paths:
       - path: /
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)